templates/web/src/sdk.ts.twig

```diff
@@ -9,6 +9,55 @@ type Headers = {
     [key: string]: string;
 }
 
+type RealtimeResponse = {
+    type: "error"|"event"|"connected"|"response";
+    data: RealtimeResponseAuthenticated|RealtimeResponseConnected|RealtimeResponseError|RealtimeResponseEvent;
+}
+
+type RealtimeRequest = {
+    type: "authentication";
+    data: RealtimeRequestAuthenticate;
+}
+
+type RealtimeResponseEvent = {
+    event: string;
+    channels: string[];
+    timestamp: number;
+    payload: unknown;
+}
+
+type RealtimeResponseError = {
+    code: number;
+    message: string;
+}
+
+type RealtimeResponseConnected = {
+    channels: string[];
+    user?: object;
```

Comment:
Can we type `user` objects too?

Reply:
This object will not be exposed to any public method yet and is only used internally. This will be added when we add complete support for response models.